database/buildSchema/29_review_response.sql

```diff
@@ -1,15 +1,13 @@
 -- review response
 
+CREATE TYPE public.review_response_decision as ENUM ('Approve', 'Decline');
+
 CREATE TABLE public.review_response (
 	id serial primary key,
-	application_response_id integer references public.application_response(id),
-	review_decision public.review_decision,
 	comment varchar,
-	trigger public.trigger
-);
-
--- TRIGGER (Listener) on Review_response table
-CREATE TRIGGER review_response_trigger AFTER INSERT OR UPDATE OF trigger ON public.review_response
-FOR EACH ROW
-WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'Processing')
-EXECUTE FUNCTION public.add_event_to_trigger_queue();
\ No newline at end of file
+	decision public.review_response_decision,
+	review_question_assignment_id integer references public.review_question_assignment(id),
+	application_response_id integer references public.application_response(id),
+	review_id integer references public.review(id),
+    "timestamp" timestamp with time zone
```

Comment:
```suggestion
    "timestamp" timestamp default current_timestamp
```
Trying to keep timezone out of back-end completely. Should be a presentation feature only, don't you think? Also handy to have default, means don't have to specify it when creating.

Reply:
Also, don't think field name "timestamp" needs to be quoted.

Reply:
Hmm, it's a bit of a habit, I am actually surprised that field names like status and timestamp work without being quoted, is this only true during creation of tables ? Not subsequent queries ?

Reply:
Not sure, I've noticed that Postgres is fairly forgiving in queries, I've only come across a handful of reserved words, and timestamp hasn't been one of them. No biggie though, can't hurt to leave it.